Assignment6/index.html

```diff
@@ -0,0 +1,158 @@
+<html>
+    <head>
+        <link rel = "stylesheet" href = "https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity = "sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin = "anonymous">        
+     
+        <style>
+            /* put your css styles here...*/
+
+        </style>
+    </head>
+  
+    <body>
+        <h4>Treasure Hunt</h4>
+    
+        <div id = "main"></div>
+        <div class = "clearfix"></div>
+        <br>
+        <div id = "response"></div>
+        
+        <script src = "https://code.jquery.com/jquery-3.3.1.min.js" integrity = "sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin = "anonymous"></script>
+        <script src = "https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity = "sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin = "anonymous"></script>
+        <script src = "https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity = "sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin = "anonymous"></script>        
+        
+        <!--put your modal dialogs here...-->
+        <div id = "my-Modal" class = "modal" tabindex = "-1" role = "dialog">
+            <div class = "modal-dialog" role = "document">
+                <div class = "modal-content">
+                    <div class = "modal-header bg-dark text-light">
+                        <h5 class = "modal-title">Play Again?</h5>
+                    </div>
+                    <div class = "modal-body">
+                        <p>Would you like to play again?</p>
+                    </div>
+                    <div class = "modal-footer bg-dark">
+                        <button type = "button" class = "btn btn-primary" data-dismiss = "modal" id = "yes">Yes</button>
+                        <button type = "button" class = "btn btn-secondary" data-dismiss = "modal" id = "no">No</button>
+                    </div>
+                </div>
+            </div>
+        </div> 
+        
+        <script>
+        /* global $ */
+        let no;
+        // put your JavaScript code here...
+        $(document).ready(function()
+        {
+           let size = 20;
+           let value = 1;
+           //add style for rows, cols, card, and card body
+            for(let i = 0; i < Math.floor(size / 2); i++){

```

Comment:
F-2: Some of this would have been better suited for a method like updateView. I know you didn't visit the link in the tip from the assignment. It could have helped!